tree.go

```diff
@@ -448,6 +448,12 @@ func (n *InternalNode) CreatePath(path []byte, stemInfo stemInfo, comms []*Point
 			// unknown node.
 			n.children[path[0]] = Empty{}
 		case extStatusAbsentOther:
+			if len(comms) == 0 {
+				return comms, fmt.Errorf("missing commitment for stem %x", stemInfo.stem)
+			}
```

Comment:
Added multiple `if len(comms) == 0` here and below. I'm just trying to prevent any panic situation when we do the `comm[1:]` reslicings.